.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request 💡
+about: Suggest a new idea for the project.
+title: ''
+labels: enhancement
+assignees: ''
+---
+
+[comment]: <> (If the feature happens in multiple SDK, open it in the integration-guide instead: https://github.com/meilisearch/integration-guides/issues)
+[comment]: <> (This is not an exhaustive model but a help. Nothing is mandatory)
+
+**Description**
+Brief explanation of the feature.
+
+**Basic example**
+If the proposal involves something new or a change, include a basic example. Omit this section if it's not applicable.
```

Comment:
```suggestion
If the proposal involves something new or a change, include a basic example.
```

Reply:
No need to say it's not mandatory since you already say "If the proposal involves something new or a change"